ipwb/util.py

```diff
@@ -144,6 +150,10 @@ def setIPWBReplayIndexPath(cdxj):
 
 def getIPWBReplayIndexPath():
     ipfsJSON = readIPFSConfig()
```

Comment:
> The ipfs config needs to be read to check if the ipwb config option are present. If not, options are written and the variable refreshed with the new ipfs config containing the ipwb config options.

Well in that case the second attempt to `readIPFSConfig` should be inside the conditional (so that it is not called again unnecessarily) like this:

```python
def getIPWBReplayIndexPath():
    ipfsJSON = readIPFSConfig()
    if 'Ipwb' not in ipfsJSON:
        setIPWBReplayConfig(IPWBREPLAY_IP, IPWBREPLAY_PORT)
        ipfsJSON = readIPFSConfig()
```